agent/node.go

```diff
@@ -584,8 +589,8 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 			m, err := manager.New(&manager.Config{
 				ForceNewCluster: n.config.ForceNewCluster,
 				ProtoAddr: map[string]string{
-					"tcp":  n.config.ListenRemoteAPI,
-					"unix": n.config.ListenControlAPI,
+					"remote":  n.config.ListenRemoteAPI,
```

Comment:
This doesn't look right. The whole `ProtoAddr` concept here is a little weird. Both APIs should be mountable to `unix` or `tcp`. This makes the mistake of separating control and remote, which is only an implementation detail of docker.